OpenOversight/config.py

```diff
@@ -15,6 +15,9 @@
 UNLABELLED_UPLOADS = 'uploads/'
 ALLOWED_EXTENSIONS = set(['png', 'jpg', 'jpeg', 'mpeg', 'mp4'])
 
+# pagination
+OFFICERS_PER_PAGE = 2
```

Comment:
how about an OFFICERS_PER_PAGE of os.environ.get('OFFICERS_PER_PAGE') or 20, so that we can override it for testing and still have a large set of results in prod?